contracts/VaultV3.vy

```diff
@@ -1840,27 +1974,49 @@ def maxMint(receiver: address) -> uint256:
     @return The maximum amount of shares that can be minted.
     """
     max_deposit: uint256 = self._max_deposit(receiver)
+    # Con't convert max uint
+    if max_deposit == MAX_UINT256:
```

Comment:
Can we move this check to `_convert_to_shares`? Maybe replicate it in `_convert_to_asset`?

Reply:
https://github.com/yearn/yearn-vaults-v3/pull/179/commits/1262aa2317dde98b832369858d2873220599b7d2